lua/nvim-tree/explorer/node-builders.lua

```diff
@@ -4,6 +4,10 @@ local watch = require "nvim-tree.explorer.watch"
 
 local M = {
   is_windows = vim.fn.has "win32" == 1,
+  is_wsl = (function()
+    local proc_version = io.open("/proc/version"):read("*a")
```

Comment:
Blows up when not found. Set to `io.open("/procccccccccc/version")`.

Need to test the results of io.open.

```
Error executing vim.schedule lua callback: ...vt-dev/site/pack/packer/start/packer.nvim/lua/packer.lua:339: Vim(lua):E5108: Error executing lua ...ree.lua.doanamo/lua/nvim-tree/explorer/node-builders.lua:8: attempt to index a nil value
stack traceback:
        ...ree.lua.doanamo/lua/nvim-tree/explorer/node-builders.lua:8: in function <...ree.lua.doanamo/lua/nvim-tree/explorer/node-builders.lua:7>
        ...ree.lua.doanamo/lua/nvim-tree/explorer/node-builders.lua:10: in main chunk
        [C]: in function 'require'
        ...nvim-tree.lua.doanamo/lua/nvim-tree/explorer/explore.lua:4: in main chunk
        [C]: in function 'require'
        ...rt/nvim-tree.lua.doanamo/lua/nvim-tree/explorer/init.lua:9: in main chunk
        [C]: in function 'require'
        ...acker/start/nvim-tree.lua.doanamo/lua/nvim-tree/core.lua:2: in main chunk
        [C]: in function 'require'
        ...rt/nvim-tree.lua.doanamo/lua/nvim-tree/renderer/init.lua:1: in main chunk
        [C]: in function 'require'
        ...packer/start/nvim-tree.lua.doanamo/lua/nvim-tree/lib.lua:3: in main chunk
        [C]: in function 'require'
        ...ack/packer/start/nvim-tree.lua.doanamo/lua/nvim-tree.lua:4: in main chunk
        [C]: in function 'require'
        /tmp/nvt-dev.lua:32: in function 'setup'
        [string ":lua"]:1: in main chunk
        [C]: in function 'cmd'
        ...vt-dev/site/pack/packer/start/packer.nvim/lua/packer.lua:339: in function ''
        vim/_editor.lua: in function <vim/_editor.lua:0>
stack traceback:
        [C]: in function 'cmd'
        ...vt-dev/site/pack/packer/start/packer.nvim/lua/packer.lua:339: in function ''
        vim/_editor.lua: in function <vim/_editor.lua:0>
```